charts/scalardb/values.yaml

```diff
@@ -123,6 +123,29 @@ scalardb:
     # -- The log level of Scalar DB
     dbLogLevel: INFO
 
+  # -- The database.properties is created based on the values of scalardb.storageConfiguration by default. If you want to customize database.properties, you can override this value with your database.properties.
+  databaseProperties: |
+    # Comma separated contact points. For DynamoDB, the region is specified by this parameter.
+    # The value of scalardb.storageConfiguration.contactPoints is set by default.
+    scalar.db.contact_points={{ default .Env.SCALAR_DB_CONTACT_POINTS "" }}
+
+    # Port number for all the contact points. Default port number for each database is used if empty.
+    # The value of scalardb.storageConfiguration.contactPort is set by default.
+    scalar.db.contact_port={{ default .Env.SCALAR_DB_CONTACT_PORT "" }}
+
+    # Credential information to access the database. For Cosmos DB, username isn't used. For DynamoDB, AWS_ACCESS_KEY_ID is specified by the username and AWS_ACCESS_SECRET_KEY is specified by the password.
+    # The value of scalardb.storageConfiguration.username is set by default.
+    scalar.db.username={{ default .Env.SCALAR_DB_USERNAME "" }}
+    # The value of scalardb.storageConfiguration.password is set by default.
+    scalar.db.password={{ default .Env.SCALAR_DB_PASSWORD "" }}
+
+    # Storage implementation. "cassandra" or "cosmos" or "dynamo" or "jdbc" or "grpc" can be set. Default storage is "cassandra".
+    # The value of scalardb.storageConfiguration.storage is set by default.
+    scalar.db.storage={{ default .Env.SCALAR_DB_STORAGE "" }}
+
+  # -- Secret name that includes credentials. Each secret key is passed to Pod as environment variables using envFrom.
+  credentialsSecretName: ""
```

Comment:
If we create the `Secret` resource includes some credentials and specify the `Secret` resource name in this value, each `Secret Key` is passed to Pod as environment variables.

We can use those environment variables for templating of the database.properties.